src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,138 @@
+package ui;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.DukeException;
+import duke.Storage;
+import task.Task;
+import task.Tasklist;
+
+public class Ui {
+
+    private static String breakline = "____________________________________________________________";
```

Comment:
You could consider declaring this variable final if the value is not going to be modified.